manifest_staging/charts/secrets-store-csi-driver/templates/keep-crds-upgrade-hook.yaml

```diff
@@ -98,7 +98,7 @@ spec:
       restartPolicy: Never
       containers:
       - name: crds-keep
-        image: "{{ .Values.linux.crds.image.repository }}:{{ .Values.linux.crds.image.tag }}"
+        image: {{ template "csidriver.image" .Values.linux.crds.image }}
```

Comment:
`template "csidriver.image"` always returns an image (with sha/tag). Do we need `.Values.linux.crds.image` as default?